### ISSUE TITLE
Treegrid is not fully mouse and keyboard operable

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria-practices#1483]
https://w3c.github.io/aria-practices/examples/treegrid/treegrid-1.html:

> Enter: Performs default action associated with row or cell that has focus, e.g. opens message or navigate to link

## Open the message
There is no way for mouse users to open the message.

Keyboard users can press Enter to open the message in the first two examples (when the focus is on the line), but not in the last example because the focus cannot be set on the line.

## Open the link
Keyboard users can navigate to the link in different ways (with Tab and with arrow keys; Tab navigation seems redundant to me). In both cases the link cannot be activated with the keyboard
